integration-libs/segment-refs/schematics/add-segment-refs/__snapshots__/index_spec.ts.snap

```diff
@@ -0,0 +1,29 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Spartacus segment-refs schematics: ng-add Segment-refs feature eager loading should import appropriate modules 1`] = `
+"import { NgModule } from '@angular/core';
+import { SegmentRefsModule } from "@spartacus/segment-refs";
+
+@NgModule({
+  declarations: [],
+  imports: [
+    SegmentRefsModule
+  ]
+})
+export class SegmentRefsFeatureModule { }
+"
+`;
+
+exports[`Spartacus segment-refs schematics: ng-add Segment-refs feature general setup should add the feature using the lazy loading syntax 1`] = `
+"import { NgModule } from '@angular/core';
+import { SegmentRefsModule } from "@spartacus/segment-refs";
+
+@NgModule({
+  declarations: [],
+  imports: [
+    SegmentRefsModule
+  ]
+})
+export class SegmentRefsFeatureModule { }
```

Comment:
For consistency with other features, let's import statically the RootModule in the FeatureModule, e.g. in this case `SegmentRefsRootModule`.

`SegmentRefsModule` would be used only for lazy loading, however we have nothing to lazy load now, so let's remove `SegmentRefsModule` (usage and source code file) and let's leave only `SegmentRefsRootModule` imported here

Reply:
done